Content.Client/GameObjects/Components/Command/CommunicationsConsoleBoundUserInterface.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Content.Client.Command;
+using Content.Shared.GameObjects.Components.Command;
+using Robust.Client.GameObjects.Components.UserInterface;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Client.GameObjects.Components.Command
+{
+    public class CommunicationsConsoleBoundUserInterface : BoundUserInterface
+    {
+        [ViewVariables]
+        private CommunicationsConsoleMenu _menu;
+
+        public bool CountdownStarted { get; private set; }
+
+        public int Countdown => _expectedCountdownTime == null
+            ? 0 : Math.Max((int)(_expectedCountdownTime.Value.Subtract(DateTime.Now)).TotalSeconds, 0);
+        private DateTime? _expectedCountdownTime;
```

Comment:
You should be using a `TimeSpan` for this, gotten from `IGameTiming.CurTime`.

Reply:
Done.